includes/admin/class-sensei-editor-wizard.php

```diff
@@ -98,15 +98,30 @@ public function enqueue_admin_scripts( $hook_suffix ) {
 	/**
 	 * Register block patterns.
 	 *
+	 * @param WP_Screen $current_screen Current WP_Screen object.
+	 *
 	 * @access private
 	 */
-	public function register_block_patterns() {
+	public function register_block_patterns( $current_screen ) {
+		$post_type = $current_screen->post_type;
+
 		// Register block pattern category.
 		register_block_pattern_category(
 			self::PATTERNS_CATEGORY,
 			array( 'label' => __( 'Sensei LMS', 'sensei-lms' ) )
 		);
```

Comment:
Based on [this](https://github.com/WordPress/wordpress-develop/blob/0a6c37ceb0a86bebf0b287ac12d200e67bbf948f/src/wp-includes/class-wp-block-pattern-categories-registry.php#L68) maybe we could register the category in `init` and let only blocks registration in the `current_screen` hook.